src/graph/transform/to_block.h

```diff
@@ -13,23 +13,34 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  *
- * @file graph/transform/to_bipartite.h
+ * @file graph/transform/to_block.h
  * @brief Functions to convert a set of edges into a graph block with local
  * ids.
  */
 
-#ifndef DGL_GRAPH_TRANSFORM_TO_BIPARTITE_H_
-#define DGL_GRAPH_TRANSFORM_TO_BIPARTITE_H_
+#ifndef DGL_GRAPH_TRANSFORM_TO_BLOCK_H_
+#define DGL_GRAPH_TRANSFORM_TO_BLOCK_H_
 
 #include <dgl/array.h>
 #include <dgl/base_heterograph.h>
 
+#include <functional>
 #include <tuple>
 #include <vector>
 
 namespace dgl {
 namespace transform {
 
+/** @brief Type of the function which maps left and right Id arrays
+ * in a MFG to new ones.
+ */
+using MappingIdsFunc =
+    std::function<std::tuple<std::vector<IdArray>, std::vector<IdArray>>(
+        const HeteroGraphPtr&, bool, int64_t, const DGLContext&,
+        const std::vector<int64_t>&, const std::vector<EdgeArray>&,
+        const std::vector<IdArray>&, const std::vector<IdArray>&,
+        std::vector<IdArray>* const, std::vector<int64_t>* const)>;
```

Comment:
why not use reference like previous arguments? legacy? or looks weird to me

Reply:
It is a requirement of google C++ style.